internal/extproc/translator/util.go

```diff
@@ -117,3 +119,14 @@ func processStop(data interface{}) ([]*string, error) {
 		return nil, fmt.Errorf("invalid type for stop parameter: expected string, []string, []*string, or nil, got %T", v)
 	}
 }
+
+// coalesce returns the value of the first non-nil pointer,
+// or the default value if all pointers are nil.
+func coalesce[T any](defaultValue T, ptrs ...*T) T {
```

Comment:
if we dont want default max tokens I can just use cmp.Or, but maxtokens is required value for anthropic